### PR TITLE
Add a CSS question about CSS stroke property

### PR DESCRIPTION
**Pull request by manojku1803**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/css-quiz.ts [hunk 1]
```diff
@@ -1197,6 +1197,17 @@ const cssQuiz = [
...
       "We can use the :last-of-type selector to target an element of a specific type within a container.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/CSS/:last-of-type/"
+  },
+  {
+    Question:
+      "Which of the below CSS property is used to add a stroke in the text ?",
+    Answer: ":text-stroke",
+    Distractor1: ":text-transform",
+    Distractor2: ":text-decoration",
+    Distractor3: ":None of the above",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1207]: None of the following answers need to have a `:` in the beginning

```suggestion
    Answer: "text-stroke",
    Distractor1: "text-transform",
    Distractor2: "text-decoration",
    Distractor3: "None of the above",
```

File: src/data/css-quiz.ts [hunk 2]
```diff
@@ -1197,6 +1197,17 @@ const cssQuiz = [
...
+    Explanation:
+      "The text-stroke is a CSS property that is used to add a stroke in the text. It is also supported by using WebKit.",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1209]: I don't think we need that last sentence.
```suggestion
      "text-stroke is a CSS property that is used to add a stroke in the text.",
```

File: src/data/css-quiz.ts [hunk 3]
```diff
@@ -1197,6 +1197,17 @@ const cssQuiz = [
...
+    Link: "https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke#:~:text=The%20%2Dwebkit%2Dtext%2Dstroke,of%20strokes%20for%20text%20characters."
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1210]:
```suggestion
    Link: "https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke"
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by manojku1803** [comment]
Check the changes @jdwilkin4

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**